private/bufpkg/bufcheck/buflint/internal/buflintv1/buflintv1.go

```diff
@@ -47,6 +47,10 @@ import "github.com/bufbuild/buf/private/bufpkg/bufcheck/internal"
 //  * COMMENTS
 //  * UNARY_RPC
 //
+//  A new category is also added, which is similar to OTHER but differently named TODO:
+//
+//  * SPECIAL
```

Comment:
This is just temporary

Reply:
Sweet, will we instead make it possible for a rule to be uncategorized?